Gemfile

```
@@ -79,9 +75,8 @@ group :development do
   gem 'web-console'
 
   # Linters
-  gem 'rubocop', require: false
+  gem 'rubocop', require: false 
```

```suggestion
  gem 'rubocop', require: false
```
Not needed space. Rubocop should automatically rise an offense against this.